src/obb_demo/processor.cljs

```diff
@@ -55,6 +55,7 @@
       {:game (deployed-game)}
       (-> (assoc game-data :actions actions)
           (assoc :original-actions actions)
+          (assoc :history (concat (or (:history game-data) (:history game)) [actions]))
```

Comment:
I think `get` here would also be a little more readable. Nothing relevant though.